static/js/components/NewsFeed.jsx

```diff
@@ -67,6 +67,60 @@ const NewsFeedItem = ({ item }) => {
       /* eslint-disable react/display-name */
       entryTitle = "New source";
       break;
+    case "classification":
+      EntryAvatar = () => (
+        <Avatar
+          alt="C"
+          size={32}
+          style={{
+            width: 32,
+            height: 32,
+            backgroundColor: "#361e66",
+            color: "white",
+            fontSize: "10px",
+          }}
+        >
+          C
+        </Avatar>
+      );
+      entryTitle = "New classification";
+      break;
+    case "spectrum":
+      EntryAvatar = () => (
+        <Avatar
+          alt="sp"
+          size={32}
+          style={{
+            width: 32,
+            height: 32,
+            backgroundColor: "#aa6622",
+            color: "white",
+            fontSize: "10px",
+          }}
+        >
+          SP
+        </Avatar>
+      );
+      entryTitle = "New spectrum";
+      break;
+    case "photometry":
+      EntryAvatar = () => (
+        <Avatar
+          alt="ph"
+          size={32}
+          style={{
+            width: 32,
+            height: 32,
+            backgroundColor: "#bbbbaa",
+            color: "white",
+            fontSize: "10px",
+          }}
+        >
+          PH
+        </Avatar>
+      );
+      entryTitle = "New photometry";
```

Comment:
Since we are still interested in knowing who uploaded the photometry / spectra, I think it's OK to use the same avatar generation as is done for users (first and last initial)

Reply:
Probably just return the uploader info like we do with comment author info for this.

Reply:
done